scripts/integration-test

```diff
@@ -5,6 +5,9 @@ export CATTLE_DEV_MODE=yes
 export CATTLE_SERVER_URL="https://$(ip route get 8.8.8.8 | awk '{print $7}'):443"
 export CATTLE_BOOTSTRAP_PASSWORD="admin"
 export CATTLE_FEATURES="harvester=false"
+export CATTLE_CHART_DEFAULT_BRANCH="dev-v2.8"
```

Comment:
Should we document those as "temporary"?

Reply:
This is should be fine since we will remove it in this week.